boards/nordic/nrf52dk_base/src/lib.rs

```diff
@@ -412,7 +412,8 @@ pub unsafe fn setup_board(
         None
     };
 
-    // Initialize AC
+    // Initialize AC using AIN0 (P0.02) as VIN+ and VIN- as AIN1 (P0.03)
```

Comment:
After looking, it turns out that P0.03 does not exist on the nrf52840_dongle. Given that we are already hardcoding the pin selection for users, could we perhaps instead choose pins P0.29 and P0.31 (AIN5 and AIN7 respectively)? These pins are mapped on all 3 nordic board that Tock supports (and that are initialized in this file)

Reply:
The only valid assignments for VIN- for the nrf52832 are AIN0 or AIN1. Since P0.02 is on the dongle too, I chose VIN+ = AIN5 and VIN- = AIN0 (there were some other small register differences between the two chips but they seemed small enough to ignore safely, and they didn't change register alignments). 
By the way, the nrf52840 dongle has the same nrf52840 chip, but instead of being a full board it's a smaller dongle you plug into your computer?

Reply:
That's reasonable, thanks for making the change. And yep pretty much!